src/app/portlets/dot-edit-page/shared/models/page-view.model.ts

```diff
@@ -0,0 +1,10 @@
+import { Layout} from './layout.model';
+import { Page } from './page.model';
+
+export interface PageView {
+    layout: Layout;
+    page: Page;
+    containers?: any
```

Comment:
this is a string[]

Reply:
the string[] type are the ones inside layout,